api/Element.json

```diff
@@ -3153,28 +3260,59 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Element/requestFullscreen",
           "support": {
             "webview_android": {
-              "version_added": null
+              "version_added": true,
+              "alternative_name": "webkitRequestFullscreen"
             },
             "chrome": {
-              "version_added": null
+              "version_added": true,
+              "alternative_name": "webkitRequestFullscreen"
             },
             "chrome_android": {
-              "version_added": null
+              "version_added": true,
+              "alternative_name": "webkitRequestFullscreen"
             },
             "edge": {
-              "version_added": null
+              "version_added": true
             },
             "edge_mobile": {
               "version_added": null
             },
-            "firefox": {
-              "version_added": null
-            },
-            "firefox_android": {
-              "version_added": null
-            },
+            "firefox": [
+              {
+                "version_added": "47",
+                "notes": "Before Firefox 44, Firefox incorrectly allowed elements inside a <code><frame></code> or an <code><object></code> to request, and to be granted, fullscreen. In Firefox 44 and onwards this has been fixed: only elements in the top-level document or in an <code><iframe></code> with the <code>allowfullscreen</code> attribute can be displayed fullscreen.",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "full-screen-api.unprefix.enabled",
+                    "value_to_set": "true"
+                  }
+                ]
+              },
+              {
+                "version_added": "9",
+                "alternative_name": "mozRequestFullScreen"
+              }
+            ],
+            "firefox_android": [
+              {
+                "version_added": "47",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "full-screen-api.unprefix.enabled",
+                    "value_to_set": "true"
+                  }
+                ]
+              },
+              {
+                "version_added": "9",
+                "alternative_name": "mozRequestFullScreen"
+              }
+            ],
             "ie": {
-              "version_added": null
+              "version_added": "11",
+              "alternative_name": "msRequestFullscreen"
             },
             "opera": {
               "version_added": null
```

Comment:
This is:
```json
"opera": [
  {
    "version_added": "15",
    "prefix": "webkit"
  },
  {
    "version_added": "12",
    "version_removed": "15",
    "prefix": "o"
  }
]
```
Same for `opera_android`, expect the switch from `oRequestFullscreen` to `webkitRequestFullscreen` [happened in version 14](https://dev.opera.com/blog/opera-14-for-android-is-out/).